GenderPayGap.WebUI/Controllers/Guidance/EightWaysGuidanceController.cs

```diff
@@ -74,7 +74,7 @@ public static PaginationPages GetPagesInThisGuidance(IUrlHelper helper, HttpRequ
                 },
                 new PaginationPage
                 {
-                    Title = "Do people get �stuck� at certain levels within your organisation?",
+                    Title = "Do people get ‘stuck’ at certain levels within your organisation?",
```

Comment:
IntelliJ automatically did this - I think it updated the characters used for the single quotes (even though it doesn't look like it)!